tslib/src/functions/ps-report-transformer/mocks/answers.ts

```diff
@@ -0,0 +1,43 @@
+import moment from 'moment'
+import { Answer } from '../../../functions-throttled/ps-report-2-pupil-data/models'
```

Comment:
might be better to pull this back into the root models folder?

Reply:
did you look into this @jon-shipley ?

Reply:
I did an initial investigation, but more time would be needed to determine the right fit here.